packages/@aws-cdk/aws-eks/lib/user-data.ts

```diff
@@ -13,6 +13,8 @@ export function renderAmazonLinuxUserData(clusterName: string, autoScalingGroup:
 
   const extraArgs = new Array<string>();
 
+  extraArgs.push(`--apiserver-endpoint '${cluster.clusterEndpoint}'`);
+  extraArgs.push(`--b64-cluster-ca '${cluster.clusterCertificateAuthorityData}'`);
```

Comment:
Can you reference some docs around these flags?

Reply:
What sort of docs are you looking for?  
Is just point to https://github.com/awslabs/amazon-eks-ami/blob/911119250416f460bbd60acd86e4763c006749fa/files/bootstrap.sh#L20-L21 sufficient ?

Reply:
Yeap thats great. Just wanted to see somewhere those options are declared.